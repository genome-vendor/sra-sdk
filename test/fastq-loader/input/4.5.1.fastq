@BILLIEHOLIDAY_1_FC20F3DAAXX
G12312312312312312312312312312312312
@BILLIEHOLIDAY_1_FC21F3DAAXX
G12112312312312312312 312312312312312
@BILLIEHOLIDAY_1_FC22F3DAAXX
G1232231231231231231 2312312312312312
@BILLIEHOLIDAY_1_FC23F3DAAXX
G12313312312312312312312312312312312
@BILLIEHOLIDAY_1_FC24F3DAAXX
G12312112312312312312312312312312312
