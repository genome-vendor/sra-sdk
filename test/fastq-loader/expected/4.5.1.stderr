latf-load warn: file="4.5.1.fastq"
latf-load warn:syntax error, unexpected fqASCQUAL, expecting fqBASESEQ or fqCOLORSEQ
latf-load warn: file="4.5.1.fastq"
latf-load warn:syntax error, unexpected fqASCQUAL, expecting fqBASESEQ or fqCOLORSEQ
latf-load err: data invalid while closing table - Too many bad records: records: 5, bad records: 2, bad records percentage: 0.40, allowed percentage: 0.20
latf-load warn: path not found while removing directory within database module - table CS_SEQUENCE failed to drop temporary column 'TMP_KEY_ID'
latf-load err: data invalid while closing table - accession="obj" errors="2" status="failure"
latf-load err: data invalid while closing table - load failed
